src/app/page/component/content-animation/content-animation.component.ts

```diff
@@ -70,22 +70,24 @@ export class ContentAnimationComponent implements OnChanges, OnDestroy {
     }
   }
 
-  onAnimationCreated(anim: any) {
+  onAnimationCreated(anim: AnimationItem): void {
     this.anmViewItem = anim as AnimationItem;
   }
 
   private init(): void {
-    // TODO
```

Comment:
Thanks for resolving this TODO! There's another reference to `e31_1` in `src/app/_tests/mock.ts`. This is optional but would you be up for fixing it too?